tests/sentry/api/endpoints/test_project_rule_details.py

```diff
@@ -80,6 +80,45 @@ def test_with_null_environment(self):
         assert response.data["id"] == six.text_type(rule.id)
         assert response.data["environment"] is None
 
+    def test_with_filters(self):
+        self.login_as(user=self.user)
+
+        project = self.create_project()
+
+        conditions = [
+            {"id": "sentry.rules.conditions.first_seen_event.FirstSeenEventCondition"},
+            {"id": "sentry.rules.filters.issue_occurrences.IssueOccurrencesFilter", "value": 10},
```

Comment:
Dang, naming is hard. I had to read this a couple of times to try and come up with a good understanding of what would happen with these 2 rules applied together. Is this meant to say, when I see a new issue alert me after seeing 10 events under that issue?

Reply:
Truth be told this rule would technically never fire because there is no way it can pass the condition of being first seen and pass the filter of having occurred 10 times. However this specific test is just making sure that if a rule is created with filters and conditions that they are split up properly in the json response. If the impossible rule is a bit weird to look at I can change the first condition to be something else, though the test doesn't change much.

Reply:
Not a problem, but it is a bit weird :-)